libs/wire-api/src/Wire/API/Routes/Version/Wai.hs

```diff
@@ -61,6 +61,13 @@ parseVersion req = do
 looksLikeVersion :: Text -> Bool
 looksLikeVersion version = case T.splitAt 1 version of (h, t) -> h == "v" && T.all isDigit t
 
+-- | swagger-delivering end-points are not disableable: they should work for all versions.
+requestIsDisableable :: Request -> Bool
+requestIsDisableable (pathInfo -> path) = case path of
+  ["api", "swagger-ui"] -> False
```

Comment:
It think `api-internal` is missing. See e.g. https://staging-nginz-https.zinfra.io/api-internal/swagger-ui/brig

Reply:
Err nevermind, internal apis are not versioned :)

Reply:
no, wait.  didn't we want to make internal apis non-versioned?